res/circlearrow.svg

```diff
@@ -1,3 +1,6 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```

Comment:
Thanks.
Who can give this a final review then? I had pushed a new commit to address some of @julienw's feedback.